packages/bot-web-ui/src/components/dashboard/joyrideConfig.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+
+const TourGuideComponent = data => {
+    const { Header, Para } = data;
+    return (
+        <div>
+            <h1>{Header}</h1>
+            <p>{Para}</p>
+        </div>
+    );
+};
+
+export default {
+    DashBorardSteps: [
+        {
+            target: 'body',
+            content: (
+                <TourGuideComponent
+                    Header={'Start with a tempplate'}
+                    Para={
+                        'Load a template containing the Martingale, D’Alembert, or Oscar’s Grind strategy, and modify it as you wish.'
+                    }
+                />
+            ),
+            showProgress: true,
+            showSkipButton: true,
+            spotlightClicks: true,
+            disableBeacon: true,
```

Comment:
@rupato-deriv I believe would be better to avoid the same fragments multiple times. Just create one object and pass it to the same places as a spread object.

Reply:
```
const other_common_properties = {
            showProgress: true,
            showSkipButton: true,
            spotlightClicks: true,
            disableBeacon: true,
 };
```
```suggestion
...other_common_properties,
```